composer.json

```
@@ -30,27 +30,27 @@
         "doctrine/persistence": "^2.2 || ^3.0",
         "jean85/pretty-package-versions": "^1.3.0 || ^2.0.1",
         "psr/log": "^1.0 || ^2.0 || ^3.0",
-        "symfony/config": "^5.4 || ^6.2",
-        "symfony/console": "^5.4 || ^6.2",
-        "symfony/dependency-injection": "^5.4 || ^6.2",
+        "symfony/config": "^6.4 || ^7.0",
+        "symfony/console": "^6.4 || ^7.0",
+        "symfony/dependency-injection": "^6.4 || ^7.0",
         "symfony/deprecation-contracts": "^2.1 || ^3.0",
-        "symfony/doctrine-bridge": "^5.4.19 || ^6.2",
-        "symfony/framework-bundle": "^5.4 || ^6.2",
-        "symfony/http-kernel": "^5.4 || ^6.2",
-        "symfony/options-resolver": "^5.4 || ^6.2"
+        "symfony/doctrine-bridge": "^6.4 || ^7.0",
+        "symfony/framework-bundle": "^6.4 || ^7.0",
+        "symfony/http-kernel": "^6.4 || ^7.0",
+        "symfony/options-resolver": "^6.4 || ^7.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^11.0",
-        "doctrine/data-fixtures": "^1.3",
+        "doctrine/data-fixtures": "^1.7",
         "phpunit/phpunit": "^9.5.5",
         "psalm/plugin-symfony": "^5.0",
-        "symfony/browser-kit": "^5.4 || ^6.2",
-        "symfony/form": "^5.4 || ^6.2",
-        "symfony/phpunit-bridge": "^6.2",
-        "symfony/security-bundle": "^5.4 || ^6.2",
-        "symfony/stopwatch": "^5.4 || ^6.2",
-        "symfony/validator": "^5.4 || ^6.2",
-        "symfony/yaml": "^5.4 || ^6.2",
+        "symfony/browser-kit": "^6.4 || ^7.0",
+        "symfony/form": "^6.4 || ^7.0",
+        "symfony/phpunit-bridge": "^6.4.1 || ^7.0.1",
```

```suggestion
        "symfony/phpunit-bridge": "^7.0.1",
```
I guess we can use only the latest

---

Yes, both are almost identical.